pkg/devcontainers/features/oci.go

```
@@ -0,0 +1,416 @@
+
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("blob fetch failed: %s", resp.Status)
+	}
+
+	// Buffer the blob to a temp file while computing its SHA-256 digest in one pass.
+	tmp, err := os.CreateTemp("", "kdn-feature-blob-*")
+	if err != nil {
+		return err
+	}
+	tmpPath := tmp.Name()
+	defer os.Remove(tmpPath)
```

golangci-lint (2.11.4)

[error] Error return value of `os.Remove` is not checked

(errcheck)